doc-deploy-dev/action.yml

```diff
@@ -229,15 +229,19 @@ runs:
       shell: bash
       run: |
         echo "<!DOCTYPE html>" > index.html
-        echo "<meta charset=\"utf-8\">" >> index.html
+        echo "<html>" >> index.html
+        echo "  <head>" >> index.html
+        echo "    <meta charset=\"utf-8\">" >> index.html
         if [ -z "${{ inputs.redirection-title }}" ]; then
-          echo "<title>Redirecting to ${{ env.REDIRECTION_URL }}</title>" >> index.html
+          echo "    <title>Redirecting to ${{ env.REDIRECTION_URL }}</title>" >> index.html
         else
-          echo "<title>${{ inputs.redirection-title }}</title>" >> index.html
+          echo "    <title>${{ inputs.redirection-title }}</title>" >> index.html
         fi
-        echo "<meta name=\"description\" content=\"${{ inputs.redirection-description }}\">" >> index.html
-        echo "<meta http-equiv=\"refresh\" content=\"0; URL=${{ env.REDIRECTION_URL }}\">" >> index.html
-        echo "<link rel=\"canonical\" href=\"${{ env.REDIRECTION_URL }}\">" >> index.html
+        echo "    <meta name=\"description\" content=\"${{ inputs.redirection-description }}\">" >> index.html
+        echo "    <meta http-equiv=\"refresh\" content=\"0; URL=${{ env.REDIRECTION_URL }}\">" >> index.html
+        echo "    <link rel=\"canonical\" href=\"${{ env.REDIRECTION_URL }}\">" >> index.html
+        echo "  </head>" >> index.html
+        echo "</html>" >> index.html
```

Comment:
I would have probably write a template HTML file, and then replace the template tags with ``sed``. Just saying, this is a bit difficult to follow.

Reply:
That's a good point. I think we had to go this way because `sed` was not allowing us to use variable substitution inside the template.